pyproject.toml

```
@@ -13,38 +13,38 @@ repository = "https://github.com/SatelCreative/spylib"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-httpx = ">= 0.18.1 < 0.23.1"
-tenacity = "^7.0.0"
+httpx = ">= 0.18.1 <= 0.24.1"
+tenacity = "^8.2.2"
 starlette = ">= 0.15.0"
 pydantic ="^1.8.2"
 PyJWT = "^2.1.0"
 pycryptodome = "^3.10.1"
 
-fastapi = { version = ">= 0.69.0 < 0.87.0", optional = true }
+fastapi = { version = ">= 0.69.0 <= 0.99.1", optional = true }
```

How about that instead?
```suggestion
fastapi = { version = ">= 0.69.0 < 0.100.0", optional = true }
```